versions.json

```diff
@@ -2,38 +2,223 @@
+            "cd lang/c++/build; env LD_LIBRARY_PATH='TEMPLATE_CLANG_CPP_LIBRARIES:/TEMPLATE_CLANG_RUNTIME_RPATH:/TEMPLATE_BOOST_LIBCXX_ROOT/lib' make -jTEMPLATE_JOBS; make install"
+        ],
+        "external_build_steps": [
+            "cd lang/c++/build",
+            "ls -l TEMPLATE_INSTALL_PREFIX/../* ; cp -rf TEMPLATE_INSTALL_PREFIX/../* ../../ ; ls -l ../../"
+        ],
+        "fpm_directories": ["bin","include","lib"],
+        "interdependencies": [
+            "boost-libcxx",
             "clang-runtime"
-        ]
+        ],
+        "distro_dependencies": {
+            "rhel": {
+                "7": [
+                    "glibc",
+                    "libgcc"
+                ],
+                "8": [
+                    "glibc",
+                    "libgcc"
+                ],
+                "9": [
+                    "glibc",
+                    "libgcc"
+                ]
+            },
+            "debian": {
+                "11": [
+                    "libc6",
+                    "libgcc-s1"
+                ],
+                "12": [
+                    "libc6",
+                    "libgcc-s1"
+                ]
+            },
+            "ubuntu": {
+                "20.04": [
+                    "libc6",
+                    "libgcc-s1"
+                ],
+                "22.04": [
+                    "libc6",
+                    "libgcc-s1"
+                ]
+            }
+        }
     },
     "boost": {
         "git_repository": "https://github.com/boostorg/boost",
         "commitish": "boost-1.81.0",
         "version_string": "1.81.0",
-        "license": "Boost Software License 1.0",
-        "consortium_build_number": "0",
-        "package_revision": "3",
+        "license": "BSL-1.0",
+        "consortium_build_number": "1",
+        "package_revision": "0",
         "externals_root": "opt/irods-externals",
         "build_steps": [
             "git submodule update --init",
             "./bootstrap.sh --prefix=TEMPLATE_INSTALL_PREFIX --with-python=/usr/bin/python3",
             "./b2 headers",
-            "./b2 install toolset=clang --without-mpi threading=multi link=shared cxxflags='-fPIC -DBOOST_SYSTEM_NO_DEPRECATED -stdlib=libc++ -std=c++14 -nostdinc++ -ITEMPLATE_CLANG_CPP_HEADERS' linkflags='-stdlib=libc++ -Wl,-rpath,/TEMPLATE_CLANG_RUNTIME_RPATH:/TEMPLATE_BOOST_RPATH' -jTEMPLATE_JOBS"
+            "./b2 install toolset=clang --without-mpi threading=multi link=shared cxxflags='-fPIC -DBOOST_SYSTEM_NO_DEPRECATED' linkflags='-Wl,-rpath,/TEMPLATE_BOOST_RPATH' -jTEMPLATE_JOBS"
```

Comment:
Is `toolset=clang` correct?

Reply:
Yes. We are still using the clang compiler, just not libc++

Reply:
Somehow I keep forgetting that. So many moving parts.

Thanks.